mixer/template/tracespan/tracespan_handler_service.proto

```diff
@@ -227,5 +235,12 @@ message InstanceParam {
 // Optional
     string client_span = 9;
     
+    // rewrite_id is used to indicate whether create a new client span id to
+// accommodate shared span model used by Zipkin. Stackdriver separates a
+// RPC into client span and server span. To solve this incompatibility,
+// rewriting both span id of client span and parent span id of server span
+// to the same newly generated id.
+    string rewrite_id = 10;
```

Comment:
Should we name this `rewrite_client_span_id` to indicate that rewriting will only occur if `client_span` is true. Also, should this be a `bool`?

Reply:
Updated the field name to `rewrite_client_span_id`.

This is an auto generated file. Not sure exactly why they are string here...